### DISCUSSION TITLE
Fails to deliver activation code, only outputs notsignedin

### DISCUSSION THREAD

**Discussion by Rand-Random** [community/community#13979] [Copilot Conversations]
I successfully installed the extension.
I am successfully getting the message box telling me a code will be in the clipboard.
I hit Ok, but nothing happens.

Nothing in clipboard, no redirekt to a webpage for authentication, just nothing output only shows this:
![image](https://user-images.githubusercontent.com/9868626/161065616-40a35e98-26a3-4207-8c5a-1c8b66c8d28f.png)

Visual Studio 2022 17.2.0 Preview 2.1

**Comment by amrjeremyhi** [comment]
Same issue.  Admitted to Technical Preview a few hours ago and only the message box appears with no code in my clipboard.  Manually navigating to https://github.com/login/device as suggested in #13986 does not work as I have no code in my clipboard.

VS 2022 Pro 17.1.3

**Reply by kzu** [reply]
Are you also behind a proxy?

**Comment by fjferrand** [comment]
I have the same issue, I suspect it is related to proxy. Are you behind a proxy?

**Reply by amrjeremyhi** [reply]
Well I feel stupid.  I disconnected from my VPN and I get the code and the pop-up now.

**Reply by Anna1-1** [reply]
Hm I disconnected from the company VPN and still no popup/nothing copied to clipboard

**Comment by davecheney** [comment]
It is likely that your employer, internet service provider, or government may be intercepting your SSL traffic.

**Reply by Rand-Random** [reply]
Well my company uses a proxy, and if copilot would use the systems proxy it should work. Do I need to manually configure copilot to use the proxy?

**Comment by McMlok** [comment]
I have same experience. And yes I'm behind a corporate proxy which needs Authentication with NTLMv2/Kerberos. Also, our proxy works like MITM and intercept SSL payload, but I have our CA used during interception imported in Trusted CAs.

Is there place with detailed logs where I can look if problem is communication over proxy or different SSL/TLS certificate which VS Copilot extension does not trust? Thank you.

**Reply by kzu** [reply]
You can bump the logging verbosity by running VS with `devenv /log`

**Reply by McMlok** [reply]
In log I saw this error

```
LSPClient.Copilot Information: 7 : {"id":5,"error":{"code":-32603,"message":"Request signInInitiate failed with message: getaddrinfo EAI_AGAIN github.com"}}
LSPClient.Copilot Verbose: 7 : Received: {
  "jsonrpc": "2.0",
  "id": 5,
  "error": {
    "code": -32603,
    "message": "Request signInInitiate failed with message: getaddrinfo EAI_AGAIN github.com"
  }
}
LSPClient.Copilot Warning: 12 : Received error response for request 5: InternalError "Request signInInitiate failed with message: getaddrinfo EAI_AGAIN github.com": 
```
It looks like VS Copilot extension doesn't use system proxy nor proxy configured in devenv.exe.config.

**Comment by lhtuling** [comment]
Same question What a stupid question!!! Such a smart plugin does such a bad job on the authorization system

**Reply by sipi41** [reply]
I agree...

**Comment by sipi41** [comment]
any solutions?

**Comment by connorjjarvis** [comment]
Hopefully we can get the code another way as most of us can get to this website: https://github.com/login/device

**Comment by Rand-Random** [comment]
You really went ahead and released it without proxy support?
https://github.blog/2022-06-21-github-copilot-is-generally-available-to-all-developers/

**Comment by nkamenar** [comment]
This is extremely frustrating. There is a workaround that can be found [here](https://github.com/github-community/community/discussions/18132). That being said, it is still ridiculous that this used to work fine and then in a patch it broke and hasn't worked since but they went ahead with release anyway without addressing this problem that a ton of users are experiencing. I have seen hundreds of posts online talking about this problem. Really hope they actually fix this soon.

**Reply by modaye** [reply]
Have you found any feasible way? I tried most of the online tutorials, but it still didn't work

**Comment by AbdelilahZiatti** [comment]
fuuuuuuuuuck

**Comment by ricnader** [comment]
Nope...Nothing here too, i'm also trying to access from a corporate computer, not connected on vpn and in my own network.

**Comment by ciaranodonnell** [comment]
I am still having this issue on a personal device on home internet with no Proxy service using v1.94.0 of the agent
---
GitHub Copilot 1.94.0.0 (v1.94.0.0@6586dd2df)
GitHub Copilot Agent v1.94.0
[DEBUG] [agent] [2023-07-12T02:25:22.945Z] Agent service starting
[DEBUG] [agent] [2023-07-12T02:25:22.978Z] Telemetry initialized
Auth Status: NotSignedIn
NotSignedIn
NotSignedIn
